conjure-java-core/src/test/resources/example-types.yml

```diff
@@ -178,6 +178,9 @@ types:
           interface: integer
           completed: integer
           unknown: integer
+          optional: optional<string>
+          list: list<string>
+          map: map<string, string>
```

Comment:
could we just re-order this to put the `set` on an adjacent line, so we can see all the container types are in one place?

Also we should probably put these tests in thee conjure-verification server!

Reply:
I'm on my way out the door for a weekend away, would you mind making the reorder change? Otherwise if there are other issues, I can fix on Monday.
This is burning some folks working on config.